tests/wpt/web-platform-tests/webgl/conformance-1.0.3/conformance/glsl/misc/expression-list-in-declarator-initializer.html

```diff
@@ -28,12 +28,12 @@
 <meta charset="utf-8">
 <link rel="stylesheet" href="../../../resources/js-test-style.css" />
 <link rel="stylesheet" href="../../resources/glsl-feature-tests.css" />
-<script src=/resources/testharness.js></script>
-<script src=/resources/testharnessreport.js></script>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
 <script src="../../../resources/js-test-pre.js"></script>
 <script src="../../resources/webgl-test-utils.js"></script>
 <script src="../../resources/glsl-conformance-test.js"></script>
-<title></title>
+

```

Comment:
The changes to the webgl/ directory are a problem, because these are tests that are automatically imported from an upstream repository. Can we tell the linter to ignore files in this directory instead, and revert the changes to them?